#include "throw.h"

int main() {
    test_exception();
    return 0;
}